### DISCUSSION TITLE
Reactions

### DISCUSSION THREAD

**Discussion by Roshick** [procore-oss/backstage-plugin-announcements#280] [Ideas]
How would you feel about enabling the user to react to announcements? I wouldn't add a full fledged comment section, but maybe by allowing them to add emojis to announcements one could get a feeling if people are happy / unhappy about certain announcements that are made?

**Comment by kurtaking** [comment]
What about a feedback mechanism that is possibly powered via emojis? I am thinking along the lines of "Did you find this message valuable? (👍🏼 / 👎🏼 )".

**Reply by Roshick** [reply]
Yea, something like this would certainly help. If I have some spare time I will throw together some versions for testing!
